helpers/fetchApiData.ts

```diff
@@ -0,0 +1,38 @@
+import { createPagesServerClient } from "@supabase/auth-helpers-nextjs";
+import { GetServerSidePropsContext } from "next";
+
+export async function fetchApiData({
```

Comment:
I'm wrestling with passing the access token in or abstracting it away into the function. I think it's fine for now, but curious on others thoughts.

Reply:
I think I'm just going to go with `publicApiFetcher` that already exists as mentioned elsewhere.